### PR TITLE
Merge 01072021 into master

### PR DESCRIPTION
**Pull request by thanhnha121**

### CODE REVIEW

**Review by thanhnha121 — COMMENTED** [review]
Need more infomation

**Review by nhaht-work — CHANGES_REQUESTED** [review]
Remove all the console.log lines

File: example.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+var name1 = "Nha Hoang";
+
+let a = 1;
+const age = 24;
+
+a = 2;
+// age = 30;
+
+// console.log(a);
```
Comment (nhaht-work) [inline example.js:9]: Please remove this

**Comment by nhaht-work** [comment]
Remove all the console.log lines

**Pull request closed without merging**